Framework/Algorithms/src/ConvertSpectrumAxis2.cpp

```diff
@@ -217,10 +217,9 @@ MatrixWorkspace_sptr ConvertSpectrumAxis2::createOutputWorkspace(
 
   MatrixWorkspace_sptr outputWorkspace = nullptr;
   NumericAxis *newAxis = nullptr;
-  size_t size;
   if (m_toOrder) {
     // Can not re-use the input one because the spectra are re-ordered.
-    size = m_indexMap.size();
+    const size_t size = m_indexMap.size();
```

Comment:
I think you should also use this on line 226? Or get rid of the variable altogether and put `m_indexMap.size()` in the call to `create`.